src/Components/ArtworkAlert/Components/Steps.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from "react"
+
+import { Confirmation } from "Components/ArtworkAlert/Components/Steps/Confirmation"
+import { Details } from "Components/ArtworkAlert/Components/Steps/Details"
+import { Filters } from "Components/ArtworkAlert/Components/Steps/Filters"
+import { useArtworkAlertContext } from "Components/ArtworkAlert/Hooks/useArtworkAlertContext"
+
+export const Steps: FC = () => {
```

Comment:
**walkthrough**: This is where we switch between content, based on the current "step". There's no fancy transition right now. Just a snappy switch.